Serverless troubleshooting guide (#13163)
DataDog/documentation: content/en/serverless/installation/python.md

    return 'Hello from serverless!'
```

For more information on custom metric submission, see [here][4]. For additional details on custom instrumentation, see the Datadog APM documentation for [custom instrumentation][5].

If your Lambda function is running in a VPC, follow these [instructions][6] to ensure that the extension can reach Datadog API endpoints.

## Troubleshooting

> If you have trouble collecting the monitoring data after following the instructions above, refer to the [serverless monitoring troubleshooting guide][7] for help.

Reviewer: ```suggestion
If you have trouble collecting monitoring data after following the instructions above, see the [serverless monitoring troubleshooting guide][7].
```
